adapters/rest-adapter-generator/src/test/java/uk/gov/justice/services/adapters/rest/validator/BaseUriRamlValidatorTest.java

```diff
@@ -0,0 +1,108 @@
+package uk.gov.justice.services.adapters.rest.validator;
+
+import uk.gov.justice.raml.common.validator.RamlValidationException;
+import uk.gov.justice.raml.common.validator.RamlValidator;
+import uk.gov.justice.services.adapters.test.utils.builder.RamlBuilder;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class BaseUriRamlValidatorTest {
+
+    private RamlValidator validator = new BaseUriRamlValidator();
+
+    @Rule
+    public ExpectedException exception = ExpectedException.none();
+
+    @Test
+    public void shouldPassIfBaseUriContainsTheQueryPillarAndApiTier() throws Exception {
+        validator.validate(new RamlBuilder()
```

Comment:
Consider introducting a builder private method that takes in the baseURI value that can be different for each test to avoid duplicated code.